frontend/src/components/user/AccountForm.tsx

```diff
@@ -0,0 +1,322 @@
+  ) => {
+    // Remove the skill from the list of skills
+    setFieldValue(
+      "skills",
+      currentSkills.filter((s) => s.id !== skill),
+    );
+  };
+
+  const initialValues: AccountFormValues = {
+    profilePhoto: "",
+    firstName: "",
+    lastName: "",
+    dateOfBirth: "",
+    pronouns: "",
+    password: "",
+    passwordConfirm: "",
+    phoneNumber: "",
+    emergencyNumber: "",
+    skills: [],
+  };
+
+  return (
+    <Box my={12}>
+      <Formik
+        initialValues={initialValues}
+        onSubmit={(values) => {
+          const formValues = {
+            ...values,
+            profilePhoto,
+          };
+
+          // eslint-disable-next-line no-console
+          console.log(formValues);
+
+          // If submitting form for admin, omit the skills field.
+
+          // TODO: Handle form submission
+        }}
+      >
+        {({ values, setFieldValue }) => (
+          <Form>
+            <SimpleGrid columns={2} spacing={10}>
+              <FormControl>
+                <FormLabel htmlFor="firstName">First Name</FormLabel>
+                <Field
+                  as={Input}
+                  id="firstName"
+                  name="firstName"
+                  type="text"
+                  placeholder="First Name"
+                />
+              </FormControl>
+              <FormControl isRequired>
+                <FormLabel htmlFor="lastName">Last Name</FormLabel>
+                <Field
+                  as={Input}
+                  id="lastName"
+                  name="lastName"
+                  type="text"
+                  placeholder="Last Name"
+                />
+              </FormControl>
+              <FormControl>
+                <FormLabel htmlFor="dateOfBirth">Date of Birth</FormLabel>
+                <Field
+                  as={Input}
+                  id="dateOfBirth"
+                  name="dateOfBirth"
+                  type="date"
+                />
+              </FormControl>
+              <FormControl>
+                <FormLabel htmlFor="pronouns">Pronouns</FormLabel>
+                <Field
+                  as={Input}
+                  id="pronouns"
+                  name="pronouns"
+                  type="text"
+                  placeholder="She/Her, He/Him, They/Them, etc"
+                />
+              </FormControl>
+              <FormControl isRequired>
+                <Flex alignItems="center">
+                  <FormLabel htmlFor="password">Password</FormLabel>
+                  <Tooltip
+                    placement="right"
+                    label={
+                      <Flex direction="column" p={1}>
+                        <Text>• At least 8 characters </Text>
+                        <Text>• At least one capital letter</Text>
+                        <Text>• At least one number </Text>
+                        <Text>• At least one special character</Text>
+                      </Flex>
```

Comment:
```suggestion
                      <UnorderedList>
                        <ListItem>At least 8 characters </ListItem>
                        <ListItem>At least one capital letter</ListItem>
                        <ListItem>At least one number </ListItem>
                        <ListItem>At least one special character</ListItem>
                      </UnorderedList>
```
I think we can use unordered list with chakra like something above: https://chakra-ui.com/docs/components/data-display/list#unordered-list